### ISSUE TITLE
Deep Learning in R

### ISSUE THREAD

**Issue by Rashkovan** [Rashkovan/sonya#7]
Skills: Neural networks, image classification
Dataset Theme: Environment and Energy
Project: Detect deforestation patterns in satellite imagery.

Tasks:
Load image data with {magick}
Train neural networks using {tensorflow} in R
Evaluate accuracy and tune hyperparameters
